pepper-apis/docs/specification/src/pepper.yml

```diff
@@ -54,6 +54,12 @@ paths:
     $ref: 'endpoints/join.mailing.list.yml'
   /studies/{study}:
     $ref: 'endpoints/studies.study.yml'
+  /studies/{study}/invitation-check:
+    $ref: 'endpoints/studies.invitation-check.yml'
+  /studies/{study}/invitation-verify:
```

Comment:
@fat23cat I had to change the URL of this API to this. I think we need to update Angular SDK here: https://github.com/broadinstitute/ddp-angular/blob/develop/ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/invitationsServiceAgent.service.ts.

Reply:
PR is ready https://github.com/broadinstitute/ddp-angular/pull/110